src/firefly/js/ui/tap/SpatialSearch.jsx

```diff
@@ -72,6 +72,7 @@ const fldListAry= [ServerParams.USER_TARGET_WORLD_PT,SpatialRegOp,SPATIAL_TYPE,
 
 export function SpatialSearch({sx, cols, serviceUrl, serviceLabel, columnsModel, tableName, initArgs={},
                                   obsCoreEnabled:requestObsCore, capabilities, handleHiPSConnection=true,
+                                  useSIAv2= false,
```

Comment:
I understand adding this one prop(`useSIAv2`) is the most straightforward, but it will tightly couple the component to feature-specific logic. This will reduce its reusability and prevent it from being composable.